src/SDL3-Own/SDL3Texture.extension.st

```diff
@@ -1,7 +0,0 @@
-Extension { #name : 'SDL3Texture' }
-
-{ #category : '*SDL3-Own' }
-SDL3Texture >> destroy [
```

Comment:
A test reveals this was needed: `Message not understood: SDL3Texture >> #destroy`
Fits better in -Own